### PR TITLE
Add extra error handling + update plugin for initial non VIP use

### PR DESCRIPTION
**Pull request by ingeniumed**
## Description

- Update the plugin to work for non VIP use by guarding the VIP specific configs
- Add in extra error handling when the plugin file is not found, or WP errors are encountered
- Update README to add in the private directory as that was taken out accidentally

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,7 +90,9 @@ The latest version of the plugin can be downloaded from the [repository's Releas
 
 ## Usage
 
-Your governance rules are saved in `governance-rules.json`. Before diving into how it's used, a quick run down of the schema will shed light on how it works.
+Your governance rules are saved in `governance-rules.json` in [your private folder][wpvip-private-dir]. Before diving into how it's used, a quick run down of the schema will shed light on how it works.
+
+Note: The [private folder][wpvip-private-dir] is only supported on VIP sites, and while using [`vip dev-env`](https://docs.wpvip.com/how-tos/local-development/use-the-vip-local-development-environment/).
```
Comment (alecgeatches) [inline README.md]:
```suggestion
Note: The [private folder][wpvip-private-dir] is only supported on VIP sites, or while using [`vip dev-env`](https://docs.wpvip.com/how-tos/local-development/use-the-vip-local-development-environment/) locally.
```

Comment (ingeniumed) [reply README.md]: Committed it in

File: governance/settings/settings.php [hunk 2]
```diff
@@ -108,11 +108,17 @@ public static function render() {
...
-		$governance_error      = false;
-
-		if ( is_wp_error( $governance_rules ) ) {
-			$governance_error = $governance_rules->get_error_message();
+		if ( is_wp_error( $governance_rules_json ) ) {
+			$governance_error      = $governance_rules_json->get_error_message();
+			$governance_rules_json = '';
+		} else {
+			$governance_rules = GovernanceUtilities::get_parsed_governance_rules();
+			$governance_error = false;
```
Comment (alecgeatches) [inline governance/settings/settings.php]: Super minor - I'd pull this line out before the `if` (line 110) to make it a little more obvious that `$governance_error` is scoped at root level and `false` is the default. PHP has weird variable hoisting that makes this work though, so it's fine if unchanged.

**Review by alecgeatches — DISMISSED** [review]
Good to go! Glad you found and fixed some error issues.

**Review by alecgeatches — APPROVED** [review]

**Pull request merged**